win32ss/user/winsrv/consrv/console.c

```diff
@@ -3,7 +3,7 @@
  * PROJECT:         ReactOS Console Server DLL
  * FILE:            win32ss/user/winsrv/consrv/console.c
  * PURPOSE:         Console Management Functions
- * PROGRAMMERS:     G� van Geldorp
+ * PROGRAMMERS:     Gé van Geldorp
```

Comment:
We do have an encoding change here. Is it to the good (towards UTF8 wo BOM)? Or to the bad (away from UTF8woBOM)?

Reply:
Good, according to Hermès's today commit.

Reply:
Github did that, from 8...something to UTF-8